src/components/UpcomingEventCard/EventCard.style.js

```diff
@@ -53,10 +53,22 @@ const UpcomingEventsWrapper = styled.div`
 		background: ${props => props.theme.secondaryColor};
 	}
 
+	.swiper-pagination {
+		background: #fff;
```

Comment:
Looking good @MukulKolpe 🎉  Can you please use our theme color here instead of the code. We'll be good to go then.